### ISSUE TITLE
[Bug Report] aria-expanded in v-expansion-panel

### ISSUE THREAD

**Issue by xdoo** [vuetifyjs/vuetify#11071]
### Environment
**Vuetify Version:** 2.2.21
**Vue Version:** 2.6.10
**Browsers:** Chrome 80.0.3987.163
**OS:** Linux x86_64

### Steps to reproduce
1. go to https://vuetifyjs.com/en/components/expansion-panels/#accordion
2. open the dev tools
3. open in the usage sample the first item
4. select this item with the element inspector to see the generated html structure.  

### Expected Behavior
According to https://www.w3.org/TR/wai-aria-practices-1.1/#accordion the "header button element has aria-expanded set to true". 

### Actual Behavior
The parent div element has a property aria-expanded set to true. This doesn't work for all screenreaders. 

### Reproduction Link
<a href="https://codepen.io/clausstraube/pen/BaoamWP?editors=1010" target="_blank">https://codepen.io/clausstraube/pen/BaoamWP?editors=1010</a>

**Comment by xdoo** [comment]
![Bildschirmfoto von 2020-04-09 17-14-29](https://user-images.githubusercontent.com/712911/78910710-a1ae1180-7a85-11ea-910e-eeddc4527967.png)

**Comment by jacekkarczmarczyk** [comment]
There are few more points that are not implemented now:
- The title of each accordion header is contained in an element with role button
- The accordion header button element has aria-controls set to the ID of the element containing the accordion panel content
- Each accordion header button is wrapped in an element with role heading that has a value set for aria-level that is appropriate for the information architecture of the page.
etc

**Comment by jerrto** [comment]
Is it going to be fixed in next release?

**Comment by johnleider** [comment]
resolved by #14014

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
